adhocracy4/categories/forms.py

```diff
@@ -30,6 +30,15 @@ def show_categories(self):
         return module_has_categories
 
 
+class CategoryAliasForm(forms.ModelForm):
```

Comment:
I changed this from ModuledashboardForm to the ModelForm (also for the labels) and remobed the second commit. The rest stayed the same.
The changes for the dashboard component are only added in mB now. We need to add it to a4 when we want to have it in a+ as well, and then we probably need to make sure it still works in opin and overwrite there.